### PR TITLE
Feat/move edit state pack

### PR DESCRIPTION
**Pull request by logan-anderson**
A possible fix for #2081

### CODE REVIEW

File: packages/tinacms/package.json [hunk 1]
```diff
@@ -36,14 +36,14 @@
     "lodash.has": "^4.5.2",
     "lodash.merge": "^4.6.2",
     "lodash.set": "^4.3.2",
-    "next": "9.4.2",
```
Comment (logan-anderson) [inline packages/tinacms/package.json:39]: @tinacms/maintainers Why is this a dep?

File: packages/tinacms/src/auth/TinaCloudProvider.tsx [hunk 2]
```diff
@@ -95,12 +96,12 @@ export const AuthWallInner = ({
...
-                // setEdit(false)
-                // setActiveModal(null)
+                setEdit(false)
+                setActiveModal(null)
 
                 // This is a temp fix
-                setEditing(false) // set editing just sets the local storage
-                window.location.reload()
+                // setEditing(false) // set editing just sets the local storage
+                // window.location.reload()
```
Comment (Enigmatical) [inline packages/tinacms/src/auth/TinaCloudProvider.tsx:104]: Minor nitpick, but I think it might be safe to just remove the commented lines in this PR.

**Review by Enigmatical — COMMENTED** [review]
I guess the ultimate question is does this resolve the context issue?

**Comment by logan-anderson** [comment]
> I guess the ultimate question is does this resolve the context issue?


@Enigmatical 

Yes it does! 😄

**Comment by jeffsee55** [comment]
Closing for now, we may want to revive this if/when we run into the same problem from the studio work

**Pull request closed without merging**